pyccel/ast/bitwise_operators.py

```diff
@@ -68,9 +66,14 @@ class PyccelBitOperator(PyccelOperator):
     arg2: PyccelAstNode
         The second argument passed to the operator
     """
-    __slots__ = ()
-
-    def _set_dtype(self):
+    _shape = ()
+    _rank = 0
+    _order = None
+    __slots__ = ('_dtype','_precision')
+
+    def _set_order(self):
+        pass
+    def _calculate_dtype(self, *_args):
```

Comment:
Is there a reason for calling this variable `_args` instead of simply `args` (without underscore)?

Reply:
I didn't know the underscore was reserved for private variables 👍🏻